tensorboard/plugins/graph/tf_graph/tf-graph-scene.ts

```diff
@@ -44,6 +44,8 @@ class TfGraphScene2 extends LegacyElementMixin(PolymerElement) {
   colorBy: string;
   @property({type: Boolean})
   traceInputs: boolean;
+  @property({type: Boolean})
+  extractNodes: boolean;
```

Comment:
nit: throughout this PR, wdyt about renaming `extractNodes` to `autoExtractNodes`?

In case we build out more node extraction logic in the future, I think `autoExtractNodes` would be a clearer name to avoid confusion with manual node extraction, for example.

Reply:
Yup, will do.

Reply:
Will do.